src/app/file-manager/transferables-grid/transferables-grid.component.ts

```diff
@@ -26,6 +27,7 @@ import { Observable } from 'rxjs/Observable';
 export class TransferablesGridComponent implements OnInit, AfterViewInit {
 
   static isExporting: Boolean = false;
```

Comment:
Do these fields have to be static? Can't any object that needs their values get them from the component? We shouldn't need to use global state to store these values.